spec/support/page_objects/eligiblity_interface/start.rb

```diff
@@ -0,0 +1,15 @@
+module PageObjects
+  class Start < PageObjects::Base
```

Comment:
```suggestion
  class Start < Base
```

As we're within the `PageObjects` module we shouldn't need this here.